integration/components/UserInfo.tsx

```diff
@@ -98,6 +103,18 @@ export default class UserInfo extends React.Component<Record<string, never>, Use
         }
     }
 
+    listDevices() {
+        try {
+            this.setState({loading: true});
+            IronWeb.user.listDevices().then((result) => {
+                this.setState({loading: false, deviceList: result.result});
+            });
+        } catch (e: any) {
+            this.setState({loading: true});
```

Comment:
Should this be `false` since something went wrong and it's no longer loading?

Reply:
Yep